src/maks.kulykov/DatabaseManager.java

```diff
@@ -0,0 +1,48 @@
+package maks.kulykov;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseManager {
+    private String db = "mydb";
+    private String userName = "postgres";
+    private String password = "postgres";
+
+    private String cmdDB;
+    private String cmdUserName;
+    private String cmdPassword;
+
+    public DatabaseManager(String db, String userName, String password) {
+        this.cmdDB = db;
+        this.cmdUserName = userName;
+        this.cmdPassword = password;
+    }
+
+    public String checkCredentials() {
+        String resp = "";
```

Comment:
this short name is not clear, don't shy  use the full words

Reply:
and if I don't know what do you planned to implement I won't guess what is resp means. Naming should help to understand the logic. I'd name it like 'wrongCredential'

Reply:
+